gamemode/modules/hungermod/sv_hungermod.lua

```diff
@@ -1,27 +1,34 @@
+
 local function HMPlayerSpawn(ply)
-	ply:setSelfDarkRPVar("Energy", 100)
+	ply:setSelfDarkRPVar("Energy", GAMEMODE.Config.maxhunger)
+	ply.LastHungerUpdate = CurTime()
 end
 hook.Add("PlayerSpawn", "HMPlayerSpawn", HMPlayerSpawn)
 
 local function HMThink()
-	if not GAMEMODE.Config.hungerspeed then return end
+	if !GAMEMODE.Config.hungerspeed then return end
+	local time = GAMEMODE.Config.hungertimer or 10
 
 	for k, v in pairs(player.GetAll()) do
-		if v:Alive() and (not v.LastHungerUpdate or CurTime() - v.LastHungerUpdate > 10) then
+		if v:Alive() and (not v.LastHungerUpdate or CurTime() - v.LastHungerUpdate > time) and !GAMEMODE.Config.hungerexcludes[v:Team()] then
 			v:hungerUpdate()
+		elseif GAMEMODE.Config.hungerexcludes[v:Team()] then
```

Comment:
This solution is way too specific for @Hazbelll's needs and wouldn't benefit **anyone** else